docs/developers-guide/substreams-sink-files.md

```diff
@@ -0,0 +1,214 @@
+2023-01-09T07:45:02.563-0800 INFO (substreams-sink-files) reading substreams manifest {"manifest_path": "substreams.yaml"}
+2023-01-09T07:45:02.563-0800 INFO (substreams-sink-files) starting pprof server {"listen_addr": "localhost:6060"}
+2023-01-09T07:45:04.041-0800 INFO (pipeline) computed start block {"module_name": "jsonl_out", "start_block": 0}
+2023-01-09T07:45:04.042-0800 INFO (substreams-sink-files) ready, waiting for signal to quit
+2023-01-09T07:45:04.045-0800 INFO (substreams-sink-files) setting up sink {"block_range": {"start_block": 0, "end_block": "None"}, "cursor": {"Cursor":"","Block":{}}}
+2023-01-09T07:45:04.048-0800 INFO (substreams-sink-files) starting new file boundary {"boundary": "[0, 10000)"}
+2023-01-09T07:45:04.049-0800 INFO (substreams-sink-files) boundary started {"boundary": "[0, 10000)"}
+2023-01-09T07:45:04.049-0800 INFO (substreams-sink-files) starting stats service {"runs_each": "2s"}
+2023-01-09T07:45:06.052-0800 INFO (substreams-sink-files) substreams sink stats {"progress_msg_rate": "0.000 msg/s (0 total)", "block_rate": "650.000 blocks/s (1300 total)", "last_block": "#1299 (a0f0f283e0d297dd4bcf4bbff916b1df139d08336ad970e77f26b45f9a521802)"}
+```
+
+One bundle of data is created for every 10K blocks during the sink process.
+
+To view the files the `substreams-sink-files` tool generates navigate into the directory you used for the output path. The directory referenced in the example points to the `localdata/out` directory. List the files in the output directory using the standard `ls` command to reveal the files created by the `substreams-sink-files` tool.
+
+```bash
+...
+0000000000-0000010000.jsonl	0000090000-0000100000.jsonl	0000180000-0000190000.jsonl
+0000010000-0000020000.jsonl	0000100000-0000110000.jsonl	0000190000-0000200000.jsonl
+0000020000-0000030000.jsonl	0000110000-0000120000.jsonl	0000200000-0000210000.jsonl
+0000030000-0000040000.jsonl	0000120000-0000130000.jsonl	0000210000-0000220000.jsonl
+0000040000-0000050000.jsonl	0000130000-0000140000.jsonl	0000220000-0000230000.jsonl
+0000050000-0000060000.jsonl	0000140000-0000150000.jsonl	0000230000-0000240000.jsonl
+0000060000-0000070000.jsonl	0000150000-0000160000.jsonl	0000240000-0000250000.jsonl
+0000070000-0000080000.jsonl	0000160000-0000170000.jsonl	0000250000-0000260000.jsonl
+0000080000-0000090000.jsonl	0000170000-0000180000.jsonl
+...
+```
+
+The block range spanned by the example is from block 0000090000 to block 0000180000. The blocks contain all the lines received for the full 10K of processed blocks by default. The block range is controlled by using the `--file-block-count` flag.
```

Comment:
> The block range spanned by the example is from block 0000090000 to block 0000180000.

I'm not sure I quite get this line. Does it refer to a single bundle range or the full range the example ran from? Also, formatting wise, unless we explicitly refer to the filename which has leading 0, we should refer to range using "natual" number.

Reply:
I'm saying it's confusing because you talk about range 90,000 - 180,000 here, but the output of the command shows block outside of the mentioned range.

Reply:
I see. I traced the steps and VSCode was cutting off the view of the output while I was writing. I misinterpreted the blocks that were generated on my machine. 

I've reintroduced the sentence using the block numbers 0000000000 to 0000260000 to reflect the full range of generated files/blocks.

Is this better?

Thank you for the clarification.

Reply:
<img width="501" alt="Screen Shot 2023-01-16 at 2 58 31 PM" src="https://user-images.githubusercontent.com/108995541/212777478-08e6fe12-aac6-45cf-a737-bfd234648c9b.png">

This is what I'm referring to with VSCode. It's a plugin to render the markdown. 

I think we got it figured out though and this is good to know. Thanks.

Reply:
Yeah perfect I see now!

Reply:
Awesome! Thank you again for the clarification. Sorry for any confusion!!